### PR TITLE
fixed bug in CoarseDropout 'apply_to_keypoints'

### PR DESCRIPTION
**Pull request by domef**

### CODE REVIEW

File: albumentations/augmentations/dropout/coarse_dropout.py [hunk 1]
```diff
@@ -167,14 +167,14 @@ def _keypoint_in_hole(self, keypoint: KeypointType, hole: Tuple[int, int, int, i
     def apply_to_keypoints(
         self, keypoints: Sequence[KeypointType], holes: Iterable[Tuple[int, int, int, int]] = (), **params
     ) -> List[KeypointType]:
-        result = []
+        result = set(keypoints)
         for hole in holes:
             remaining_keypoints = []
             for kp in keypoints:
                 if not self._keypoint_in_hole(kp, hole):
                     remaining_keypoints.append(kp)
```
Comment (Dipet) [inline albumentations/augmentations/dropout/coarse_dropout.py:175]: I think better just remove keypoint from the set

**Comment by Dipet** [comment]
Thank you for finding this terrible bug. Please add this test case into tests.
```python
@pytest.mark.parametrize(
    ["keypoints", "expected_keypoints", "holes"],
    [
        [[(50, 50, 0, 0), (75, 75, 0, 0)], [], [(40, 40, 60, 60), (70, 70, 80, 80), (10, 10, 20, 20)]],
        [[(50, 50, 0, 0), (75, 75, 0, 0)], [(75, 75, 0, 0)], [(40, 40, 60, 60), (10, 10, 20, 20)]],
        [[(50, 50, 0, 0), (75, 75, 0, 0)], [(50, 50, 0, 0)], [(70, 70, 80, 80), (10, 10, 20, 20)]],
        [[(50, 50, 0, 0), (75, 75, 0, 0)], [(50, 50, 0, 0), (75, 75, 0, 0)], [(10, 10, 20, 20)]],
    ]

)
def test_coarse_dropout_remove_keypoints(keypoints, expected_keypoints, holes):
    t = A.CoarseDropout()
    result_keypoints = t.apply_to_keypoints(keypoints, holes)

    assert set(result_keypoints) == set(expected_keypoints)```

**Comment by Dipet** [comment]
https://github.com/albumentations-team/albumentations/blob/master/tests/test_transforms.py

**Comment by Dipet** [comment]
Oh, no, Better to this one https://github.com/albumentations-team/albumentations/blob/master/tests/test_keypoint.py

**Review by Dipet — APPROVED** [review]

**Comment by bartoszptak** [comment]
Hi, I installed the master branch from Github to fix the bug. But now, I obtain the error:
```
    transformed = self.transforms(image=image, keypoints=annos[:, 1:3], labels=annos[:, 0], shapes=annos[:, 3:5])
  File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/core/composition.py", line 210, in __call__
    data = t(**data)
  File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/core/transforms_interface.py", line 118, in __call__
    return self.apply_with_params(params, **kwargs)
  File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/core/transforms_interface.py", line 131, in apply_with_params
    res[key] = target_function(arg, **dict(params, **target_dependencies))
  File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/augmentations/dropout/coarse_dropout.py", line 170, in apply_to_keypoints
    result = set(keypoints)
TypeError: unhashable type: 'numpy.ndarray'
```

Any idea? The error doesn't occur when the key point is not covered by CoarseDropout.

**Comment by JunkyByte** [comment]
> Hi, I installed the master branch from Github to fix the bug. But now, I obtain the error:
> 
> ```
>     transformed = self.transforms(image=image, keypoints=annos[:, 1:3], labels=annos[:, 0], shapes=annos[:, 3:5])
>   File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/core/composition.py", line 210, in __call__
>     data = t(**data)
>   File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/core/transforms_interface.py", line 118, in __call__
>     return self.apply_with_params(params, **kwargs)
>   File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/core/transforms_interface.py", line 131, in apply_with_params
>     res[key] = target_function(arg, **dict(params, **target_dependencies))
>   File "/home/bartosz/miniconda3/envs/count/lib/python3.10/site-packages/albumentations/augmentations/dropout/coarse_dropout.py", line 170, in apply_to_keypoints
>     result = set(keypoints)
> TypeError: unhashable type: 'numpy.ndarray'
> ```
> 
> Any idea? The error doesn't occur when the key point is not covered by CoarseDropout.

@bartoszptak you are probably using `np.ndarray` as keypoints, you should be using list of tuples.
Check https://albumentations.ai/docs/getting_started/keypoints_augmentation/ the code works correctly with such keypoints.

**Pull request merged**